### DISCUSSION TITLE
Change BUTTON_INNER based on value

### DISCUSSION THREAD

**Discussion by bphunter1972** [Textualize/textual#2312] [Ideas]
From #1934, wouldn't this make a whole lot more sense in _toggle_button.py?

```python
BUTTON_INNER: dict[bool, str] = {True: "X", False: " "}
```

I thought that checkboxes were simply broken, but instead it was that the bold attribute simply doesn't stand out enough at smaller fonts.

Then of course in _button():
```python
        return Text.assemble(
            (self.BUTTON_LEFT, side_style),
            (self.BUTTON_INNER[self.value], button_style),
            (self.BUTTON_RIGHT, side_style),
        )
```

**Comment by davep** [comment]
Could you perhaps give a visual example or two to illustrate the issue you're seeing?
